apps/sms/views/conversation/js/conversation.js

```diff
@@ -1910,9 +1920,16 @@ var ConversationView = {
     }
     params.items.push(subjectItem);
 
-    // If we are on a thread, we can call to SelectMessages
+    // If we are on a thread, we can call to SelectMessages and add recipients
     if (Navigation.isCurrentPanel('thread')) {
       params.items.push({
+        l10nId: 'addRecipients-label',
+        method: () => {
+          this.updateDraft();
+          this.initiateNewMessage({ draft: this.draft });
```

Comment:
OK now I think we should not do that; simply we should just create a new draft here, using Compose.getContent(), etc.